indigo_api/serializers.py

```diff
@@ -104,16 +105,12 @@ class AmendmentEventSerializer(serializers.Serializer):
 
 
 class RepealSerializer(serializers.Serializer):
-    """ Serializer matching :class:`cobalt.RepealEvent`, for use describing
-    the repeal on a published document.
-    """
+    """ Describes the repeal event for a work. """
+    # Matches :class:`cobalt.RepealEvent`
 
-    date = serializers.DateField()
-    """ Date that the repeal took place """
-    repealing_title = serializers.CharField()
-    """ Title of repealing document """
-    repealing_uri = serializers.CharField()
-    """ FRBR URI of repealing document """
+    date = serializers.DateField(help_text="Effective date of the repeal.")
```

Comment:
@longhotsummer do these need to be translated?

Reply:
no

Reply:
Reasoning: the schema is generated once in English and then made available. So changing based on the user's language I don't think make sense (and I don't think is supported by drf-spectacular, redoc, or swagger).